Releases.md

```diff
@@ -4,6 +4,7 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 
 | Version                | Feature level                                                                                                                | First available in (Windows 10)                          | Redistributable                                                                           |
 | ---------------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| [1.8.0](#directml-180) | [DML_FEATURE_LEVEL_5_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_5_0) | TBD                                                      | [Microsoft.AI.DirectML.1.8.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.8.0) |
 | [1.7.0](#directml-170) | [DML_FEATURE_LEVEL_4_1](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_4_1) | TBD                                                      | [Microsoft.AI.DirectML.1.7.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.7.0) |
 | [1.6.0](#directml-160) | [DML_FEATURE_LEVEL_4_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_4_0) | TBD                                                      | [Microsoft.AI.DirectML.1.6.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.6.0) |
```

Comment:
DML 1.6 is available in Win11, so we can update this now as well.

Reply:
Adding column. 21H2 is [evidently](https://www.osgwiki.com/wiki/Windows_version_history) still VB.